xbmc/video/dialogs/GUIDialogVideoManagerExtras.cpp

```diff
@@ -55,7 +55,6 @@ bool CGUIDialogVideoManagerExtras::OnMessage(CGUIMessage& message)
       {
         AddVideoExtra();
       }
-      break;
```

Comment:
I would keep this `break;`, although not having it does not cause any problems atm. But if another case gets added in the future, one can easily forget to reintroduce the break here.

Reply:
ok I didn't mean to remove it.